bundles/org.openhab.binding.logreader/src/main/java/org/openhab/binding/logreader/internal/filereader/FileTailer.java

```diff
@@ -34,6 +35,7 @@
     private final Logger logger = LoggerFactory.getLogger(FileTailer.class);
 
     private Tailer tailer;
+    private final ExecutorService executor = Executors.newFixedThreadPool(1);
```

Comment:
This threadpool is never shutdown so I expect if this class is removed it will still keep the threadpool maybe either simple start the thread itself or make sure the threadpool is shutdown when this class is removed. Probably on stop it can be this class is reused and then the executor can't be shutdown there.